src/templates/next/render/renderPageContent.ts

```diff
@@ -0,0 +1,24 @@
+import { renderComponent } from "~/templates/next/render"
+import { IsomerComponent } from "~/types"
+
+export const renderPageContent = ({
+  content,
+  LinkComponent,
+}: {
+  content: IsomerComponent[]
+  LinkComponent: any
+}) => {
+  let isTextOnRight = false
+  return content.map((component) => {
+    if (component.type === "infopic") {
+      console.log(isTextOnRight)
```

Comment:
nit: this one should be removed?

Reply:
Removed in https://github.com/isomerpages/isomer-components/pull/84/commits/4fe54d7bb4390cd14064afe250617388ed13fd50!